apis/kots/v1beta1/license_types_test.go

```diff
@@ -52,6 +52,7 @@ metadata:
 spec:
   licenseID: test-license-id
   licenseType: trial
+	customerID: test-customer-id
```

Comment:
indentation is wrong here

```suggestion
  licenseType: trial
  customerID: test-customer-id
```